src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -193,15 +193,12 @@ private static OpenApiResponses GetOpenApiResponses(MethodInfo method, EndpointM
         foreach (var annotation in eligibileAnnotations)
         {
             var statusCode = annotation.Key.ToString(CultureInfo.InvariantCulture);
-            var (type, contentTypes) = annotation.Value;
+            var (_, contentTypes) = annotation.Value;
             var responseContent = new Dictionary<string, OpenApiMediaType>();
 
             foreach (var contentType in contentTypes)
             {
-                responseContent[contentType] = new OpenApiMediaType
-                {
-                    Schema = OpenApiSchemaGenerator.GetOpenApiSchema(type)
-                };
+                responseContent[contentType] = new OpenApiMediaType();
```

Comment:
Swashbuckle will now read this information and populate the `Schema`, is that how it will work?

Reply:
Yep! See https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/2441.